virtusize/build.gradle.kts

```diff
@@ -71,7 +71,7 @@ android {
 
 dependencies {
     api(project(":virtusize-core"))
-    api(libs.virtusize.auth) {
+    api(project(":virtusize-auth")) {
         exclude(group = "com.virtusize.android", module = "virtusize-core")
```

Comment:
Could you try removing this to see if it works correctly without virtusize-core being transient in the virtusize-auth module?